.github/workflows/reusable-integration-test-v1.yml

```diff
@@ -7,7 +7,6 @@ on:
         type: string
         description: 'The reference of https://github.com/RetricSu/godwoken-kicker.git'
         required: false
```

Comment:
```
required: true
default: "compatibility-changes"
```

Suggest using default value and change the below `ref: ${{ inputs.kicker_ref || 'compatibility-changes' }}` to `ref: ${{ inputs.kicker_ref }}`

Reply:
If this workflow is **NOT** triggered by `workflow_call` event, `inputs.kicker_ref` is empty.

Reply:
Oh, I see. You are correct.